databricks/koalas/tests/test_groupby.py

```diff
@@ -748,9 +747,11 @@ def test_shift(self):
                        pdf.groupby(['b'])['a'].shift().sort_index(), almost=True)
         self.assert_eq(kdf.groupby(['a', 'b'])['c'].shift().sort_index(),
                        pdf.groupby(['a', 'b'])['c'].shift().sort_index(), almost=True)
-        self.assert_eq(kdf.groupby(['b'])[['a', 'c']].shift(periods=-1, fill_value=0).sort_index(),
-                       pdf.groupby(['b'])[['a', 'c']].shift(periods=-1, fill_value=0).sort_index(),
-                       almost=True)
+        # TODO: seems like a pandas' bug when fill_value is not None?
+        # self.assert_eq(
+        #     kdf.groupby(['b'])[['a', 'c']].shift(periods=-1, fill_value=0).sort_index(),
+        #     pdf.groupby(['b'])[['a', 'c']].shift(periods=-1, fill_value=0).sort_index(),
+        #     almost=True)
```

Comment:
This one, same. We can just test when pandas version is not 1.0.0. Can you describe why it looks like a bug in pandas?

Reply:
- 0.24.0 <= pandas < 1.0.0
```python
>>> pdf.groupby(['a', 'b']).shift(periods=-1, fill_value=0).sort_index()
          a  b   c
0.061934  2  2   9
0.348846  0  0   0
0.417891  2  2  16
0.442334  3  4  36
0.453143  3  3  25
0.523323  2  2  16
0.523910  2  2  16
0.574407  0  0   0
0.587845  0  0   0
0.639976  3  4  36
0.670847  1  1   1
0.675415  1  1   4
0.798390  0  0   0
0.832881  2  2   9
0.853427  1  1   4
0.856181  1  1   4
0.904274  3  3  25
0.997218  1  1   1
```

- pandas >= 1.0.0
```python
>>> pdf.groupby(['a', 'b']).shift(periods=-1, fill_value=0).sort_index()
Traceback (most recent call last):
...
KeyError: 0.0
```

Reply:
> We can just test when pandas version is not 1.0.0

thanks! i fixed with following this way